fenix/app/src/main/java/org/mozilla/fenix/BrowsingModeBinding.kt

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix
+
+import android.view.Window
+import android.view.WindowManager
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.distinctUntilChangedBy
+import kotlinx.coroutines.withContext
+import mozilla.components.lib.state.helpers.AbstractBinding
+import org.mozilla.fenix.browser.browsingmode.BrowsingMode
+import org.mozilla.fenix.components.AppStore
+import org.mozilla.fenix.components.appstate.AppState
+import org.mozilla.fenix.theme.ThemeManager
+import org.mozilla.fenix.utils.Settings
+
+/**
+ * Binding to react to Private Browsing Mode changes in AppState.
+ *
+ * @param appStore AppStore to observe state changes from.
```

Comment:
```suggestion
 * Binding to react to Private Browsing Mode changes in [AppState].
 *
 * @param appStore [AppStore] to observe state changes from.
```
Let's make these references linkable(or able to jump to definition) where possible. This will also help us refactor things if we ever rename them.